arch/risc-v/src/esp32c3/Kconfig

```diff
@@ -261,6 +262,29 @@ config ESP32C3_SPIFLASH
 	select MTD_BYTE_WRITE
 	select MTD_PARTITION
 
+config ESP32C3_SPI_LCD
+	bool "Generic SPI LCD Module"
+	default n
+	select ESP32C3_SPI2
+	select SPI_CMDDATA
+	select ESP32C3_SPI_SWCS
+	select SPI_SWAPBYTES
+
+if ESP32C3_SPI_LCD
+
+config SPI_SWAPBYTES
+	bool
```

Comment:
Please, add a prompt message to this config.

Reply:
I encountered this issue when sending SPI commands with NuttX ST7789 drivers, which is sending 16 bits cmd code. Actually 8-bit sending is enough, but sending 16-bit cmd requires a byte swap on ESP32C3. So is this really necessary, or should I just change the drivers to send 8bit cmd instead? And the display data bytes are inverted as well, but that requires swapping from source in order to use DMA.